studio/components/layouts/DatabaseLayout/DatabaseLayout.tsx

```diff
@@ -26,7 +26,7 @@ const DatabaseLayout = ({ title, children }: PropsWithChildren<DatabaseLayoutPro
 
   const vaultExtension = meta.extensions.byId('supabase_vault')
   const isVaultEnabled = vaultExtension !== undefined && vaultExtension.installed_version !== null
-  const foreignDataWrappersEnabled = useFlag('foreignDataWrappers')
+  const wrappersExtensionExists = meta.extensions.byId('wrappers') !== undefined
```

Comment:
the menu item should still show the wrappers option as we have the appropriate empty state to handle if the wrappers extension is not installed yet